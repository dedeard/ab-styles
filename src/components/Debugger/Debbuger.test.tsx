import { render } from '@testing-library/react';

import { Debugger } from '.';

describe('Debugger', () => {
  it('should render with current screen %s', () => {
    const { container } = render(<Debugger />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
