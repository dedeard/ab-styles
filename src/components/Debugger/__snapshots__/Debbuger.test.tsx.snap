// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Debugger should render with current screen %s 1`] = `
.grid-0 {
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  z-index: 10000;
  overflow: hidden;
  pointer-events: none;
  transform-origin: 50% 0%;
}

.grid-0 .grid-3 {
  height: 100%;
  opacity: 0;
  -webkit-transition: opacity 0.2s;
  transition: opacity 0.2s;
}

.grid-0 .grid-5 {
  height: 100%;
  background: #0470F50D;
}

.grid-0 .grid-7 {
  height: 100%;
}

.grid-0 .grid-7>div {
  position: relative;
  width: 100%;
  height: 100%;
}

.grid-0 .grid-7>div::before,
.grid-0 .grid-7>div::after {
  content: '';
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background: #0470F50D;
}

.grid-2 {
  width: 100%;
  max-width: 100%;
  margin-right: auto;
  margin-left: auto;
  box-sizing: border-box;
}

@media (min-width: 1em) {
  .grid-2 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

@media (min-width: 33.75em) {
  .grid-2 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

@media (min-width: 48em) {
  .grid-2 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

@media (min-width: 64em) {
  .grid-2 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

@media (min-width: 71.25em) {
  .grid-2 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

@media (min-width: 48em) {
  .grid-2 {
    max-width: 45rem;
  }
}

@media (min-width: 64em) {
  .grid-2 {
    max-width: 60rem;
  }
}

@media (min-width: 71.25em) {
  .grid-2 {
    max-width: 71.25rem;
  }
}

.grid-4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  box-sizing: border-box;
}

@media (min-width: 1em) {
  .grid-4 {
    margin-left: -0.9375rem;
    margin-right: -0.9375rem;
  }
}

@media (min-width: 33.75em) {
  .grid-4 {
    margin-left: -0.9375rem;
    margin-right: -0.9375rem;
  }
}

@media (min-width: 48em) {
  .grid-4 {
    margin-left: -0.9375rem;
    margin-right: -0.9375rem;
  }
}

@media (min-width: 64em) {
  .grid-4 {
    margin-left: -0.9375rem;
    margin-right: -0.9375rem;
  }
}

@media (min-width: 71.25em) {
  .grid-4 {
    margin-left: -0.9375rem;
    margin-right: -0.9375rem;
  }
}

.grid-6 {
  position: relative;
  -webkit-flex: 1 0 0%;
  -ms-flex: 1 0 0%;
  flex: 1 0 0%;
  width: 100%;
  max-width: 100%;
  box-sizing: border-box;
}

@media (min-width: 1em) {
  .grid-6 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

@media (min-width: 33.75em) {
  .grid-6 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

@media (min-width: 48em) {
  .grid-6 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

@media (min-width: 64em) {
  .grid-6 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

@media (min-width: 71.25em) {
  .grid-6 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

.emotion-0 {
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  z-index: 10000;
  overflow: hidden;
  pointer-events: none;
  transform-origin: 50% 0%;
}

.emotion-0 .emotion-3 {
  height: 100%;
  opacity: 0;
  -webkit-transition: opacity 0.2s;
  transition: opacity 0.2s;
}

.emotion-0 .emotion-5 {
  height: 100%;
  background: #0470F50D;
}

.emotion-0 .emotion-7 {
  height: 100%;
}

.emotion-0 .emotion-7>div {
  position: relative;
  width: 100%;
  height: 100%;
}

.emotion-0 .emotion-7>div::before,
.emotion-0 .emotion-7>div::after {
  content: '';
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background: #0470F50D;
}

.emotion-2 {
  width: 100%;
  max-width: 100%;
  margin-right: auto;
  margin-left: auto;
  box-sizing: border-box;
}

@media (min-width: 1em) {
  .emotion-2 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

@media (min-width: 33.75em) {
  .emotion-2 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

@media (min-width: 48em) {
  .emotion-2 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

@media (min-width: 64em) {
  .emotion-2 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

@media (min-width: 71.25em) {
  .emotion-2 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

@media (min-width: 48em) {
  .emotion-2 {
    max-width: 45rem;
  }
}

@media (min-width: 64em) {
  .emotion-2 {
    max-width: 60rem;
  }
}

@media (min-width: 71.25em) {
  .emotion-2 {
    max-width: 71.25rem;
  }
}

.emotion-4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  box-sizing: border-box;
}

@media (min-width: 1em) {
  .emotion-4 {
    margin-left: -0.9375rem;
    margin-right: -0.9375rem;
  }
}

@media (min-width: 33.75em) {
  .emotion-4 {
    margin-left: -0.9375rem;
    margin-right: -0.9375rem;
  }
}

@media (min-width: 48em) {
  .emotion-4 {
    margin-left: -0.9375rem;
    margin-right: -0.9375rem;
  }
}

@media (min-width: 64em) {
  .emotion-4 {
    margin-left: -0.9375rem;
    margin-right: -0.9375rem;
  }
}

@media (min-width: 71.25em) {
  .emotion-4 {
    margin-left: -0.9375rem;
    margin-right: -0.9375rem;
  }
}

.emotion-6 {
  position: relative;
  -webkit-flex: 1 0 0%;
  -ms-flex: 1 0 0%;
  flex: 1 0 0%;
  width: 100%;
  max-width: 100%;
  box-sizing: border-box;
}

@media (min-width: 1em) {
  .emotion-6 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

@media (min-width: 33.75em) {
  .emotion-6 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

@media (min-width: 48em) {
  .emotion-6 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

@media (min-width: 64em) {
  .emotion-6 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

@media (min-width: 71.25em) {
  .emotion-6 {
    padding-left: 0.9375rem;
    padding-right: 0.9375rem;
  }
}

<div
  class="emotion-0 emotion-1"
>
  <div
    class="emotion-2 emotion-3"
  >
    <div
      class="emotion-4 emotion-5"
    >
      <div
        class="emotion-6 emotion-7"
      >
        <div />
      </div>
      <div
        class="emotion-6 emotion-7"
      >
        <div />
      </div>
      <div
        class="emotion-6 emotion-7"
      >
        <div />
      </div>
      <div
        class="emotion-6 emotion-7"
      >
        <div />
      </div>
      <div
        class="emotion-6 emotion-7"
      >
        <div />
      </div>
      <div
        class="emotion-6 emotion-7"
      >
        <div />
      </div>
      <div
        class="emotion-6 emotion-7"
      >
        <div />
      </div>
      <div
        class="emotion-6 emotion-7"
      >
        <div />
      </div>
      <div
        class="emotion-6 emotion-7"
      >
        <div />
      </div>
      <div
        class="emotion-6 emotion-7"
      >
        <div />
      </div>
      <div
        class="emotion-6 emotion-7"
      >
        <div />
      </div>
      <div
        class="emotion-6 emotion-7"
      >
        <div />
      </div>
    </div>
  </div>
</div>
`;
